result/yoda.js

```diff
@@ -0,0 +1,18 @@
+
+/*eslint no-multi-spaces: "error"*/
+
+var a =  1;
+var a =  1;
+var a =  1;
+
+if(foo   === "bar") {}
+
+a <<  b
+a <<  b
+
+var arr = [1,  2];
+
+a ?  b: c
+
+var a =  1;
```

Comment:
<!-- Comment by [SideCI](https://sideci.com) -->
__[development][eslint]__
* Error - 'a' is already defined (no-redeclare)
* Error - Multiple spaces found before '1'. (no-multi-spaces)